docs/04-operation-guides/operations/08-install-uninstall-upgrade-kyma-module.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Install, uninstall and upgrade a Kyma module
+---
+
+The Kyma project is currently in the transition phase from classic to modular Kyma. Learn how to install, uninstall and upgrade a Kyma module. To see the list of all Kyma modules, go to [Overview](/docs/01-overview/README.md).
+
+## Install a Kyma module
+
+To install a module, deploy its module manager and apply the module configuration. See the available Kyma modules with quick installation steps and links to their GitHub repositories.
+
+### Keda
```

Comment:
@kwiatekus Hi Krzysiu, could you confirm if the commands will work for the Keda module or should I change anything?